Sources/Callout/Callout.swift

```diff
@@ -0,0 +1,146 @@
+import Foundation
+import SwiftUI
+
+extension Warp {
+    /**
+     Callouts are snippets of information, drawing attention to important content.
+
+     **When to use**
+
+     Call attention to important pieces of information.
+
+    To use the iOS `Callout` View you need to provide it with
+     - a `title`
+     - an `arrowHeight`, if you don't provide this, the default value is 8 px
+     - an `arrowWidth`, if you don't provide this, the default value is 18px
+     - an `arrowEdge` where you want the Callout to appear from
+     - a `cornerRadius`, if you don't provide this, the default value is 8px
+     */
+    public struct Callout: View {
+        /// Title to present in the callout view
+        private var title: String
+
+        /// Height of the arrow to draw, default value is 8 px
+        private let arrowHeight: Double
+
+        /// Width of the arrow to draw, default value is 18 px
+        private let arrowWidth: Double
+
+
+        /// Edge where to draw the arrow, default value is `.top`
+        private var arrowEdge: Edge
+
+        /// Corner radius,
+        private let cornerRadius: Double
+
+        /// Object responsible for providing colors in different environments and variants.
+        private let colorProvider: ColorProvider
+
+        /**
+         - Parameter title: String to display in the `Callout`
+         - Parameter arrowHeight: Height of the arrow to draw, default value is 8 px
+         - Parameter arrowWidth: Width of the arrow to draw, default value is 18 px
+         - Parameter arrowEdge: Edge where to draw the arrow, default value is `.top`
+         - Parameter cornerRadius: Radius of corners, default value is 8
+         - Parameter colorProvider: ColorProvider used for styling the `Callout`, default value is read from `Config`
+         */
+        public init(
+            title: String,
+            arrowHeight: Double = 8,
+            arrowWidth: Double = 18,
+            arrowEdge: Edge = .top,
+            cornerRadius: Double = 8,
+            colorProvider: ColorProvider = Config.colorProvider
+        ) {
+            self.title = title
+            self.arrowHeight = arrowHeight
+            self.arrowWidth = arrowWidth
+            self.arrowEdge = arrowEdge
+            self.cornerRadius = cornerRadius
+            self.colorProvider = colorProvider
+        }
+
+        public var body: some View {
+            Text(title)
+                .font(from: Typography.body)
```

Comment:
font is different with the size too, seems one is using body the other caption

Reply:
and seems size small is only one liner but size default support multi line text

Reply:
I see an `enum` with computed properties in my near future. God I love `enum`s in Swift 🤓